motr/st/utils/motr_client_st.sh

```diff
@@ -41,6 +41,8 @@ m0t1fs_st_dir=$motr_st_util_dir/../../../m0t1fs/linux_kernel/st
 # Set the mode of Motr [user|kernel]
 umod=1
 
+export MOTR_CLIENT_ONLY=1
```

Comment:
It is a defined in ""m0t1fs/linux_kernel/st/m0t1fs_common_inc.sh" file.

> 738 # If this is a pure client test, set this to 1.
 739 # Otherwise, it should be 0
 740 MOTR_CLIENT_ONLY=0
 741
 742 service_eps_with_m0t1fs_get()
 743 {
 744         local lnet_nid=$(m0_local_nid_get)
 745         local service_eps=$(service_eps_get)
 746
 747         # If client only, we don't have m0t1fs nid.
 748         if [ $MOTR_CLIENT_ONLY -ne 1 ] ; then
 749                 service_eps+=("$lnet_nid:12345:33:1")
 750         fi
 751
 752         # Return list of endpoints
 753         echo "${service_eps[*]}"
 754 }
 755